about.html

```diff
@@ -0,0 +1,120 @@
+              <li><a href="index.html#program">Program</a></li>
+              <li><a href="#join">Join</a></li>
+              <li><a href="#sponsor">Sponsor</a></li>
+              <li><a href="#news">News</a></li>
+              <button class="campaignBtn">DLGC Campaign</button>
+            </ul>
+          </div>
+        </div>        
+      
+      <div class="bg-image">
+        <nav class="menuBars">
+          <div class="bar"></div>
+          <div class="bar"></div>
+          <div class="bar"></div>
+        </nav>
+      </div>
+    </header>
+
+    <section class="intro-2">
+      <h2>"Hello! Dog Lovers"</h2>
+      <div class="bold">
+        <h1>DOG LOVERS GLOBAL CONFERENCE 2022</h1>
+      </div>
+      <div class="info-2">
+        <p>The conference brings together the comunity of dog breeders, dog owners, veterinary doctors and other proffessionals who comprise the Dog Lovers affiliate network in a different country every two years.</p>
+      </div>
+      <div class="contact">
+        <p>Please contact us per email for any further questions about DLGC 2022!</p>
+        <a href="#email">glgc2022@paws.org</a>
+      </div>
+    </section>
+
+    <section class="aboutlogo">
+      <div class="logosection" id="logo">
+        <h3>Dog Lovers Global Conference 2022 Logo</h3>
+        <div class="bar-2"></div>
+        <p>The logo of DL Global Conference 2022 was decided through the logo competition from 8. March to 9. April.</p>
+        <div class="theLogo">
+          <a href="#" class="mlogo">DLGC</a>
+        </div>
+      </div>
+      <div class="past">
+        <h3>See the past DL Global Conferences</h3>
+        <div class="bar-2"></div>
+        <p>Take a look at the last two DL Global Conferences which took place in Canada and Ghana.</p>
+      </div>
+      <div class="allconcert" id="allconcert">
+        <div class="pastconcert">
+          <img class="concertimg" src="Images/concert pic.jpg">
+          <h4 class="year">2020</h4>
+          <P class="place">DL Global Conference 2020 in Canada</P> 
+        </div>
+        <div class="pastconcert">
+          <img class="concertimg" src="Images/concert pic 2.jpeg">
+          <h4 class="year">2018</h4>
+          <P class="place">DL Global Conference 2018 in Ghana</P>
+        </div>
+      </div>   
+    </section>
+
+    <section class="partner">
+      <h3>Partner</h3>
+      <div class="bar-4"></div>
+      <ul class="logos">
+        <li><img src="Images/mozilla-logo.png" alt="mozilla"></li>
+        <li><img src="Images/google logo.png" alt="google"></li>
+        <li><img src="Images/daumkakao.png" alt="daumkakao"></li>
+        <li><img src="Images/airbnb_logo.png" alt="airbnb"></li>
+      </ul>
+    </section>   
+  </main>
+  <script src="main.js"></script>
+</body>
+<footer>
+  <section class="copyrights">
+    <a href="#" class="mlogo">DLGC <br> Nigeria</a>
+    <p>2022 Dog Lovers Nigeria. <br> Some Rights Reserved</p>
+  </section>
```

Comment:
- [ ] [OPTIONAL] Your footer section is outside the `body` tag, kindly consider moving it inside the `body` before the `script` so your code is structured properly